### PR TITLE
Portland 2019 blog post events, activities, shirts, tickets

### PR DESCRIPTION
**Pull request by thatdocslady**

### CODE REVIEW

File: docs/conf/portland/2019/news/events-activities.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Every year we are amazed at the growing support of our community, and tickets have been selling out earlier each conference.
+We are now just under **80%** sold out, and judging from past events we do expect to be sold out within the next few weeks.
+
+If you're still on the fence, now is the time to decide and `get your tickets <https://ti.to/writethedocs/write-the-docs-portland-2019/>`_.
+
+Diversity tickets
+-----------------
+
+Write the Docs aims to continuosly expand our community to be more diverse, welcoming, and inclusive.
```
Comment (mxsasha) [inline docs/conf/portland/2019/news/events-activities.rst]:
```suggestion
Write the Docs aims to continuously expand our community to be more diverse, welcoming, and inclusive.
```

File: docs/conf/portland/2019/news/events-activities.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+   :tags: tickets, events, practical, shirts, portland-2019
+
+Events and Activities, T-Shirts, and Tickets Update
+===================================================
+
+We're just under two months out, and we hope everyone is getting excited!
+
+In this update, we're happy to share with you the different events and activities that will be available as a part of the conference, announce our pop-up t-shirt shop, and remind you to grab your tickets before they run out (hint: very soon!).
+
+Remember to check out our `Welcome Wagon <https://www.writethedocs.org/conf/portland/2019/welcome-wagon/>`_ guide and the `Visiting Portland <https://www.writethedocs.org/conf/portland/2019/visiting/>`_ page for help with planning your trip and making the most out of the conference.
```
Comment (mxsasha) [inline docs/conf/portland/2019/news/events-activities.rst:14]: Welcome Wagon links to a page that says:
> The 2018 conference planning group includes an Explore Portland committee. Proud locals Mo Nishiyama and Alicia Duncan will be answering questions and providing tips and information on the #wtd-conferences Slack channel as the event draws near. Feel free to ping us directly as well!

(I could update the year, but not sure if any of the other info is also outdated?)

File: docs/conf/portland/2019/news/events-activities.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Please share this campaign with your social networks and encourage folks to apply if they are interested to join us!
+
+Shirts on sale
+--------------
+
+Following last year's pilot, we are happy to announce that the `Write the Docs Portland 2019 Pop-Up Shop <https://teespring.com/wtd-portland-2019>`_ is now open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want!
+
+The campaign will run in 3-day batches until **May 30th**. We recommend to order your shirt at least 2-3 weeks in advance, to allow for timely delivery of your shirts ahead of the conference.
+
+Similarly, we will be continuing with our bag swag this year, which you can pick up at the conference.
```
Comment (mxsasha) [inline docs/conf/portland/2019/news/events-activities.rst]: Perhaps:
> we will be continuing our bag swag this year
`with` seems a bit extra

File: docs/conf/portland/2019/news/events-activities.rst [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+Events and activities
+---------------------
+
+Write the Docs is more than a conference. Every year we make sure to have a variety of activities that our attendees can enjoy, whether you want to learn something, meet new people, or share knowledge.
+
+* **Hike**. If you're in town early, join us on Saturday for our traditional `hike <https://www.writethedocs.org/conf/portland/2019/hike/>`_ is a great chance to get to know your fellow documentarians in a smaller setting. This activity is free with your conference ticket, but space is limited! We will publish a separate registration soon.
```
Comment (mxsasha) [inline docs/conf/portland/2019/news/events-activities.rst]:
```suggestion
* **Hike**. If you're in town early, join us on Saturday for our traditional `hike <https://www.writethedocs.org/conf/portland/2019/hike/>`_. This activity is a great chance to get to know your fellow documentarians in a smaller setting. The hike is free with your conference ticket, but space is limited! We will publish a separate registration soon.
```

File: docs/conf/portland/2019/news/events-activities.rst [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+* **Welcome reception.** The meet-and-greet event will be held on Sunday from *5pm-8pm*. You can pick up your badge and enjoy a couple of drinks on us. This is a great chance to meet your fellow documentarians before the conference starts.
+* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/portland/2019/lightning-talks/>`_ each day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required.
+* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/portland/2019/unconference/>`_ all day Monday and on Tuesday afternoon. Want to chat with like-minded folks about a very specific thing? This is the place. We'll have a sign-up board so you know what's happening, and supplies to help you brainstorm.
```
Comment (mxsasha) [inline docs/conf/portland/2019/news/events-activities.rst]:
```suggestion
* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/portland/2019/unconference/>`_ all day Monday and on Tuesday afternoon. Want to chat with like-minded folks about a specific thing? This is the place. We'll have a sign-up board so you know what's happening, and supplies to help you brainstorm.
```
I'd drop the "very", it feels to me like there is a minimum amount of specificity needed for an unconference session, but some are quite broad.

**Review by mxsasha — CHANGES_REQUESTED** [review]
Looks good! Left a few small comments.

File: docs/conf/portland/2019/news/events-activities.rst (hunk 2 above)
Comment (thatdocslady) [reply docs/conf/portland/2019/news/events-activities.rst:14]: Doesn't look this way from the website, I guess it was fixed already?

File: docs/conf/portland/2019/news/events-activities.rst [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+Please share this campaign with your social networks and encourage folks to apply if they are interested to join us!
+
+Shirts on sale
+--------------
+
+Following last year's pilot, we are happy to announce that the `Write the Docs Portland 2019 Pop-Up Shop <https://teespring.com/wtd-portland-2019>`_ is now open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want!
+
+The campaign will run in 3-day batches until **May 30th**. We recommend to order your shirt at least 2-3 weeks in advance, to allow for timely delivery of your shirts ahead of the conference.
+
+Similarly, we will be continuing with our bag swag this year, which you can pick up at the conference.
```
Comment (thatdocslady) [inline docs/conf/portland/2019/news/events-activities.rst]:
```suggestion
Similarly, we will be continuing our bag swag this year, which you can pick up at the conference.
```

**Comment by thatdocslady** [comment]
Thanks @mxsasha for the review, now we're just waiting for @ericholscher to hammer out the childcare registration and we can ship this :)

File: docs/conf/portland/2019/news/events-activities.rst (hunk 2 above)
Comment (mxsasha) [reply docs/conf/portland/2019/news/events-activities.rst:14]: Yes, it seems so :)

**Review by mxsasha — APPROVED** [review]

**Comment by thatdocslady** [comment]
@plaindocs the CI build seems to be taking forever... any idea what's happening? All I did was rebase =/

**Comment by thatdocslady** [comment]
Nevermind, seems to work now. Merging.

**Pull request merged**